### ISSUE TITLE
'pyopenms.pyopenms_4.PeptideAndProteinQuant' object has no attribute 'getPeptideResults'

### ISSUE THREAD

**Issue by zhuchcn** [OpenMS/OpenMS#5657]
`getPeptideResults` isn't accessible from `PeptideAndProteinQuant`, but it seems like a public method.

```
from pyopenms import PeptideAndProteinQuant
PeptideAndProteinQuant().getPeptideResults()
```
Error:
```
AttributeError: 'pyopenms.pyopenms_4.PeptideAndProteinQuant' object has no attribute 'getPeptideResults'
```

Is it possible to make it available? We would like to only get the peptide quantification but not protein, so if `getPeptideResults` is available, we can directly call quantifyPeptide from python and save it. Unless there is a better way of doing it. Btw, `getProteinResults` is also not available from pyopenms.

**Comment by timosachsenberg** [comment]
Hi, that would indeed be a good to have in pyOpenMS.

Looking at the code someone already tried to wrap it:
https://github.com/OpenMS/OpenMS/blob/cceb3d3f27a8603dab7c73609f72905f944027b9/src/pyOpenMS/pxds/PeptideAndProteinQuant.pxd#L59-L62
I guess it was not successful because the key is a AASequence object which probably needs special treatment on the wrapper side. @hroest can you take a brief look at that?

**Comment by jpfeuffer** [comment]
Yes, currently not both key and value can be user-defined/wrapped classes.
We need to investigate why autowrap has this limitation.
For just outputting to pyopenms we should just write an overload that returns std::strings as keys for those methods

**Comment by hroest** [comment]
@jpfeuffer the limitation is simply that its not implemented:

https://github.com/OpenMS/autowrap/blob/84e19fb094d628a7446120873c6819e79296bb3c/autowrap/ConversionProvider.py#L787-L790

what does work is `libcpp_map[libcpp_string, ProteinData]` while `libcpp_map[AASequence, PeptideData]` wont work

**Comment by hroest** [comment]
@jpfeuffer this PR fixes this issue and should allow such wrapping of `std::map<Widget, Widget>`: https://github.com/OpenMS/autowrap/pull/124

**Comment by jpfeuffer** [comment]
Unfortunately the PR wasn't complete. We need to wait for a fix of OpenMS/autowrap issue 143

**Comment by timosachsenberg** [comment]
Should be possible noew
